### ISSUE TITLE
Whole-measure rests throw off note spacing in other staves

### ISSUE THREAD

**Issue by mjwalter** [zolaemil/meiView#23]
In certain measures, notes are severely misplaced, often entering the following measure. This seems to be caused by the presence of an `<mRest>` element in another voice. A dramatic example of this bug can be found at [DC0302](http://duchemin-dev.haverford.edu/piece/DC0302). A simple example is in [DC0212](http://duchemin-dev.haverford.edu/piece/DC0212). Both paginated and scrolling canvases have the issue.

![screen shot 2014-07-29 at 09 51 04](https://cloud.githubusercontent.com/assets/3750681/3735739/6dbe5d52-1727-11e4-8c28-1e01c84dc7bf.png)

![screen shot 2014-07-29 at 09 54 50](https://cloud.githubusercontent.com/assets/3750681/3735817/072c8dd8-1728-11e4-8abd-7411cb6dbcdf.png)

**Comment by zolaemil** [comment]
Wrong time signature in the mei can cause this behaviour: 2/2 isn't the correct time signature for this piece, it should be 4/2.

**Comment by mjwalter** [comment]
Right. I've added a function to MEIMassage that replaces all 4/2 time signatures with 2/2 signatures (retaining the "cut time" symbol). It's untested (my libmei installation is not working) but it is a simple enough function. We will use it on our next export and massage. You may need to incorporate it into the scripts.
